pkg/ebpf/c/tracer-ebpf.c

```diff
@@ -796,8 +796,8 @@ int kprobe__tcp_get_info(struct pt_regs* ctx) {
         return 0;
     }
 
+    status->tcp_info_kprobe_status = 1;
```

Comment:
Just double-checking, is default triggered (1)?

Reply:
What do you mean by default here? This should be 0 initially and only set to 1 when this function is invoked. 

I previously had it in an if statement, but basically this is just a way to determine (we have a support case where we need to check this currently) that this kprobe runs at all.